dts/bindings/adc/st,stm32-adc.yaml

```diff
@@ -21,6 +21,21 @@ properties:
     "#io-channel-cells":
       const: 1
 
+    st,prescaler:
```

Comment:
Let's be a bit more specific for this property so users don't mix with existing STM32 timer's property 'st,prescaler'
```suggestion
    st,adc-prescaler:
```

Reply:
done